### PR TITLE
Update Rule “use-a-signature-with-a-link-when-commenting-on-a-blog/rule”

### PR DESCRIPTION
**Pull request by JackDevAU**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/use-a-signature-with-a-link-when-commenting-on-a-blog/rule.md [hunk 1]
```diff
@@ -1,30 +1,53 @@
 ---
 type: rule
-title: Do you use a signature with a link when commenting on a blog?
-uri: use-a-signature-with-a-link-when-commenting-on-a-blog
+title: Do you use signature links and monitor GitHub stars for online engagement?
+uri: use-sig-links-github-stars
```
Comment (jaydenalchin) [inline rules/use-a-signature-with-a-link-when-commenting-on-a-blog/rule.md:4]: Please also update this rule URL in its category Rules to Better Blogging 🙏

**Review by jaydenalchin — CHANGES_REQUESTED** [review]

File: rules/use-a-signature-with-a-link-when-commenting-on-a-blog/rule.md (hunk 1 above)
Comment (JackDevAU) [reply rules/use-a-signature-with-a-link-when-commenting-on-a-blog/rule.md:4]: Done - good catch 🙏

File: categories/marketing-and-video/rules-to-better-blogging.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ index:
 - add-a-featured-image-to-your-blog-post
 - where-to-find-images
 - share-every-blog-post
-- use-a-signature-with-a-link-when-commenting-on-a-blog
+- sig-links-github-stars
```
Comment (jaydenalchin) [inline categories/marketing-and-video/rules-to-better-blogging.md:30]:
```suggestion
- use-sig-links-github-stars
```

Comment (JackDevAU) [reply categories/marketing-and-video/rules-to-better-blogging.md]: that might help. Thanks!

**Review by jaydenalchin — APPROVED** [review]
LGTM 🚀

**Review by bradystroud — APPROVED** [review]

**Pull request merged**